lib/rucio/core/config.py

```diff
@@ -48,57 +61,73 @@ def add_section(section, session=None):
 
 
 @read_session
-def has_section(section, session=None):
+def has_section(section, use_cache=True, expiration_time=3600, session=None):
     """
     Indicates whether the named section is present in the configuration.
 
     :param section: The name of the section.
+    :param use_cache: Boolean if the cache should be used.
+    :param expiration_time: Time after that the cached value gets ignored.
     :param session: The database session in use.
     :returns: True/False
     """
-
-    query = session.query(models.Config).filter_by(section=section)
-
-    return True if query.first() else False
+    has_section_key = 'has_%s' % section
```

Comment:
I would find it a bit easier yo understand if the lookup keys are always the method name.
Thus
- has_section_%
- options_%
- has_option_%
...
Make it just a bit easier to read and also understand, once you just dump the full cache contents.